### DISCUSSION TITLE
formily的高性能是在于formily/reactive的Observable对象吗？

### DISCUSSION THREAD

**Discussion by EvalGitHub** [alibaba/formily#3245] [Q&A 2.x]
formily的高性能问题是在于@formily/reactive的Observable对象吗？那对于一些没有使用到formily/reactive的场景是怎么保持他的高性能的呢？例如一些常规的基础表单所涉及到的组件，不会使用到@formily/reactive，那怎么保证到的高性能呢？
望大佬指点！！！

**Comment by Elinia** [comment]
因为所有的场景都会用到 @formily/reactive。
以 react 为例，无论你使用 JSX 形式、Markup Schema 形式还是 JSON Schema 形式，你所编写的组件最终都会变成一个 ReactiveField，被套上这个 observer。 https://github.com/alibaba/formily/blob/formily_next/packages/react/src/components/ReactiveField.tsx#L113
